bears/shell/ShellCheckBear.py

```diff
@@ -17,6 +17,7 @@ class ShellCheckBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax', 'Semantic', 'Possible fails in certain situations'}
```

Comment:
`Syntax, Security, Semantics, Undefined Elements, Unused Code`

Unused Code because `echo "$(date)"                    # Useless use of echo` from `shellcheck`'s README